_posts/2017-01-11-migrating-to-puppet4.md

```diff
@@ -13,15 +13,17 @@ are a few steps needed to be taken before a Puppet-4-only release can be done:
 
 * Check the current `metadata.json` for the required Puppet version and bump it if needed
   * We already have a few modules that only support Puppet 4, in this case you can ignore this guide
-  * The minimum required Puppet Version should not be 4.0.0 but [4.6.1](https://github.com/voxpupuli/community-triage/blob/master/modules/notes/2017-01-05.md#discussion)
   * Also check if the dependencies in the `metadata.json` all have Puppet 4 support, they may have to be bumped too
   * The minimum required stdlib version has to be 4.6.0
+  * The required Puppet version should be 3.8.7 or newer
```

Comment:
The idea is that the last puppet3 release will also run the spec tests on the latest puppet version available. so I think this should be present in the metadata.json